src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/CompareVersionsStep/help-v1.html

```diff
@@ -0,0 +1,3 @@
+<p>
+    The version number string that <code>v1</code> will be compared to.
```

Comment:
If this is `v1`, maybe the text should read `v2`? IDK what makes the most sense, but same goes for `help-v2.html`:

```suggestion
    The version number string that <code>v2</code> will be compared to.
```

Reply:
Yea, it puts dust in my brain trying to write something comprehensible :D